hooks/src/index.js

```diff
@@ -43,11 +43,8 @@ options._render = vnode => {
 			hooks._pendingEffects = [];
 			currentComponent._renderCallbacks = [];
 			hooks._list.forEach(hookItem => {
-				if (hookItem._nextValue) {
```

Comment:
This can be removed as we are returning `return hookState._nextValue || hookState._value;` from useState either way